evm/README.md

```diff
@@ -197,9 +199,37 @@ $ anvil --help
 $ cast --help
 ```
 
-### Deploy Wormhole NTT
+## Message Customization
+
+See the [NttManager](../docs/NttManager.md) doc for wire format details.
+
+> Note: The size of `NttManager` is significantly higher (and very close to the contract size limit) than `NttManagerNoRateLimiting`. See [No-Rate-Limiting](#no-rate-limiting) for more detail.
+
+### NativeTokenTransfer Additional Payload
+
+Your contract can extend `NttManagerNoRateLimiting` to provide an additional payload on the `NativeTokenTransfer` message. Override the following:
+
+- `_prepareAdditionalPayload`
```

Comment:
Should be updated to `_prepareNativeTokenTransfer` now

Reply:
I was confused initially by the comment because I thought you meant the `_handleAdditionalPayload` should be updated to `_prepareNativeTokenTransfer`. So I added the suggested change above for clarity.